feat: knock test emails only go to account members (#1457)
knocklabs/docs: content/integrations/email/knock-test.mdx

layout: integrations
---

Knock comes with a built-in email channel to test your email notifications. This is a great way to get started with email notifications without needing to integrate with an external email provider.

<Callout
  type="warning"
  title="Note:"
  text={
    <>

>       This provider is for testing purposes only and is limited to send to email

Reviewer: ```suggestion
      This provider is for testing purposes only. Sending is limited to email
```
nit

>       addresses associated with the members of your Knock account.

Reviewer 2: ```suggestion
      addresses associated with the [members](/manage-your-account/managing-members) of your Knock account.
```

You'll find the Knock test channel under the **Channels and sources** page in your account settings. By default, all new Knock accounts will have the channel enabled and ready to use across all environments with no additional configuration required.

### Allowed recipients

> The Knock test channel is limited to sending emails to email addresses associated with the **members of your Knock account only**. Attempting to send emails to email addresses that are not associated with a member will result in a delivery error (`knock_recipient_not_allowed`).

Reviewer 2: ```suggestion
The Knock test channel is limited to sending emails to email addresses associated with the **[members](/manage-your-account/managing-members) of your Knock account only**. Attempting to send emails to email addresses that are not associated with a member will result in a delivery error (`knock_recipient_not_allowed`).
```

  <Accordion title="What is the limit on the number of emails I can send?">
    You can send up to 100 emails per month for free.
  </Accordion>
  <Accordion title="Who can I send emails to?">

>     You can only send emails to email addresses that are associated with the **members** of your Knock account. Attempting to send emails to email addresses that are not associated with a member will result in a delivery error.

Reviewer 2: ```suggestion
    You can only send emails to email addresses that are associated with the [**members**](/manage-your-account/managing-members) of your Knock account. Attempting to send emails to email addresses that are not associated with a member will result in a delivery error.
```
